pkg/manila/dbsync.go

```diff
@@ -79,6 +79,7 @@ func DbSyncJob(instance *manilav1.Manila, labels map[string]string, annotations
 		UserPasswordSelector: instance.Spec.PasswordSelectors.Service,
 		VolumeMounts:         GetInitVolumeMounts(dbSyncExtraMounts, DbsyncPropagation),
 		Debug:                instance.Spec.Debug.DBInitContainer,
+		Logging:              false,
```

Comment:
@gouthampacha we don't need logging using the `logging.conf` pattern in `dbsync`

Reply:
Thanks @fmount; I wanted to suggest calling this 

`UsesLoggingConf` or `SupportsLoggingConf`

to avoid confusion.. just a way to clarify that this module doesn't respect logging.conf, but uses logs...

Reply:
is `LoggingConf` a good solution? It implies a conf file that should exists but don't deny the fact we're logging anyway. If `LoggingConf` doesn't work, I'm ok to change to something like `LoggingConfSupport`, which is basically the same thing.